lib/iruby.rb

```diff
@@ -15,18 +15,22 @@
 require 'iruby/utils'
 require 'iruby/display'
 require 'iruby/comm'
-require 'iruby/session/mixin'
 
-begin
-  require 'iruby/session/cztop'
-rescue LoadError
+if ENV.fetch('IRUBY_OLD_SESSION', false)
```

Comment:
```suggestion
if ENV['IRUBY_OLD_SESSION']
```

Reply:
In this case, the default value offers no extra benefit - the absence of the String key returns `nil`, which is falsy.

Reply:
This is my intentional use of `Env.fetch`.

Reply:
It _does communicate_ intention. Carry on, don't mind my earlier input.

Reply:
I'm sorry. My previous comment is not polite.

First, thank you very much for having an interest in IRuby development.  IRuby is not mature.  IRuby needs a lot of implementation improvements to catch up on Jupyter's evolutions.

I know ENV[...] is enough here as you described.  But does the replacement of ENV.fetch with ENV[...] improves the state of IRuby itself?  I don't think so.  I wanted to spend more time for improving the design and debugging of IRuby.  It's reason why i don't fix the line you pointed out in this pull-request.

I would appreciate if you understood that.

Reply:
Thank you, @mrkn, and I apologize for how I communicated.